src/main/java/com/example/parkingapp/domain/ParkingRecord.java

```diff
@@ -0,0 +1,117 @@
+package com.example.parkingapp.domain;
+
+import com.example.parkingapp.config.AcceptedCurrencies;
+import com.example.parkingapp.config.ParkingRecordStatus;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+/**
+ * Record storing information about parking meter events
+ */
+@Entity
+@Table(name = "parking_records")
+public class ParkingRecord implements Serializable {
+    @EmbeddedId
+    private ParkingRecordID id;
+
+    @ManyToOne
+    @JoinColumn(name = "driver_id")
+    private User user;
+
+    @Column(name = "end_date")
+    private LocalDateTime endDate;
+
+    @Column(name = "pin_code_hash")
+    private String pinCode;
+
+    @Enumerated(EnumType.STRING)
+    private ParkingRecordStatus status = ParkingRecordStatus.STARTED;
+
+    @Enumerated(EnumType.STRING)
+    private AcceptedCurrencies currency;
+
+    @Column(name = "payment")
```

Comment:
kolumna nazywa się tak samo jak pole - czy to potrzebne?

Reply:
Do usunięcia